### ISSUE TITLE
Project Idea: Smart water bottle

### ISSUE THREAD

**Issue by Jmiller25** [CHS-WBL-2023/Group2#5]
This is a water bottle that is smart. It will connect with a smart watch to track your fitness and tell you when and how much to drink.

**Comment by Rocco064** [comment]
Good idea! But why cant I just drink water out of a normal water bottle.
